drivers/overlay/ov_network.go

```diff
@@ -255,11 +287,21 @@ func setHostMode() {
 }
 
 func (n *network) generateVxlanName(s *subnet) string {
-	return "vx-" + fmt.Sprintf("%06x", n.vxlanID(s)) + "-" + n.id[:5]
+	id := n.id
+	if len(n.id) > 5 {
+		id = n.id[:5]
+	}
+
+	return "vx-" + fmt.Sprintf("%06x", n.vxlanID(s)) + "-" + id
```

Comment:
Minor. Since you are touching this code, you may want to not mix string concatenation and Sprintf and do it all in one Sprintf. Up to you.